src/app/payment-hub/request-to-pay/incoming-request-to-pay/incoming-request-to-pay.component.ts

```diff
@@ -258,6 +284,10 @@ export class IncomingRequestToPayComponent implements OnInit {
         debounceTime(500),
         distinctUntilChanged(),
         tap((filterValue) => {
+          filterValue = filterValue.AlphabeticCode;
+          if('KES' === filterValue){
+            filterValue = 'KE'
```

Comment:
@simiyu17 how does the applyn filter works, is it a 'like' comparison because we have both KES and KE as currency in some of the request